opencog/scm/opencog/rule-engine/rule-engine-utils.scm

```diff
@@ -43,6 +43,7 @@
 ;; -- tv->bool -- Convert TRUE_TV to #t, anything else to #f
 ;; -- atom->number -- Convert NumberNode into its corresponding number
 ;; -- gt-zero-confidence -- Return TrueTV iff A's confidence is greater than 0
+;; -- has-dv -- Return TrueTV iff A has a Distributional Value
 ;; -- absolutely-true -- Return TrueTV iff A's TV is TrueTV
```

Comment:
This too, should probably be a distinct pull request.  For small pull reqs, ti would not matter.  For something this big, the pull req defining the thing should be distinct from the pull req that uses it.